packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -111,6 +113,10 @@ export class OvsdbClient {
 
     // Add interface and port to the bridge
     const options = ['map', [['remote_ip', remoteAddress], ['key', key]]]
+    if (password !== undefined) {
+      options[1].push(['psk', password])
+    }
+
```

Comment:
You may improve it even further by making `options` an object and then, where you use it, use [`_.toPairs`](https://devdocs.io/lodash~4/index#toPairs):
```js
options: ['map', toPairs(options)]
```

Reply:
I'll do it on the while file in a different PR.